xp-archive/client/content/Content.tsx

```diff
@@ -43,21 +45,58 @@ export const Content = () => {
     });
 
     useEffect(() => {
-        if (selectedVersion) {
-            updateContentUrl(selectedContentId ?? '', selectedLocale, selectedVersion);
-        } else if (data?.versions?.[0]) {
-            const latestVersionId = data.versions[0].versionId;
-            setSelectedVersion(latestVersionId);
-            updateContentUrl(selectedContentId ?? '', selectedLocale, latestVersionId);
+        const versionId = selectedVersion ?? data?.versions?.[0]?.versionId;
+        if (versionId) {
+            if (!selectedVersion) {
+                setSelectedVersion(versionId);
+            }
+            const newUrl = `${xpArchiveConfig.basePath}/${selectedContentId}/${selectedLocale}/${versionId}`;
+            window.history.replaceState({}, '', newUrl);
```

Comment:
Når vi endrer til å replaceState her så ødelegger vi vel muligheten for å trykke backspace?

Reply:
Ooops 😨. Bytter tilbake